src/utils/formValidationHelpers.ts

```diff
@@ -55,8 +55,23 @@ export const minLength = (min: number): Function => {
  * @param max - Maximal length of a string returned function will validate
  * @returns function validating maximal length
  */
-export const maxLength = (max: number): Function => {
+export const maxLength = (max: number) => {
   return (string: string): boolean => {
     return !string || string.length <= max;
   };
 };
+
+const dateRegex = /^\d{4}-\d{2}-\d{2}$/;
```

Comment:
That is already managed by creating a date object and testing it for NaN.

Reply:
[still optional] Ok, the only problem I have it this:

```tsx
/**
 * Date validation function.
 *
 * Accepts empty date.
 *
 * @param date - Date string
 * @returns true if valid, false otherwise
 */
export const validateDate = (date: string): boolean 
```

This sounds very general (it's general util function saying it will validate basically all dates), so soon some else can decide to use it on different places. But it's not validating the date, it's just validating NNNN-NN-NN format, not date YYYY-MM-DD. I think description or regex could be still slightly improved to make them more "compatible".

Reply:
ok